components/form/checkbox/src/index.js

```diff
@@ -0,0 +1,74 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+const FormCheckbox = ({checked, className, errorMessage, label, name, onChange, svgIcons}) => (
+  <div className={cx('sui-FormCheckbox', className)}>
+    <label className='sui-FormCheckbox-container'>
+      <input
+        checked={checked}
+        className={cx('sui-FormCheckbox-input', {'is-checked': checked})}
+        name={name}
+        onChange={onChange}
+        type='checkbox' />
+      {checked ? svgIcons.checked : svgIcons.unchecked}
+      <span className='sui-FormCheckbox-label'>{label}</span>
+    </label>
+    {errorMessage &&
+    <div className='sui-FormCheckbox-errorMessage'>
+      <span className='sui-FormCheckbox-errorMessageLabel'>{errorMessage}</span>
+    </div>
+    }
+  </div>
+)
+
+FormCheckbox.displayName = 'FormCheckbox'
+
+FormCheckbox.propTypes = {
+  /**
+   * CSS classname to apply to component's container.
+   */
+  className: PropTypes.string,
+  /**
+   * Boolean attribute that specifies if the input should be displayed as selected or not.
+   */
+  checked: PropTypes.bool,
+  /**
+   * Text to display as error message below the checkbox label.
+   */
+  errorMessage: PropTypes.string,
+  /**
+   * Text to display as a description message on the right side of the chechbox input.
+   */
+  label: PropTypes.node,
+  /**
+   * Specifies the name of the checkbox component.
+   */
+  name: PropTypes.string.isRequired,
+  /**
+   * Custom callback function to execute when the checked/uncheched status of the checkbox has changed.
+   */
+  onChange: PropTypes.func,
+  /**
+   * Allows customisation of the icons to display representing the checked/unchecked status of the component.
+   */
+  svgIcons: PropTypes.shape({
+    checked: PropTypes.node.isRequired,
+    unchecked: PropTypes.node.isRequired
+  })
+}
+
+FormCheckbox.defaultProps = {
+  svgIcons: {
+    checked: <svg className='sui-FormCheckbox-icon' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'>
```

Comment:
What about using this repo for these icons? https://github.com/SUI-Components/svg-iconset. I understand they ara not official and have to be redefined by UX/Design, but it's better to work with graphic icons that exports React components. Working directly with SVG icons (its code) can be painful.

Reply:
Yes, no problem, I can move these 2 icons to the svg-iconset repo.
Besides, do you think it's still useful to allow the customisation of the checkbox icons through a component's propert? Or not anymore? (just to know what you think about removing this `svgIcons` prop after moving the icons to the other repo).